packages/ds-healthcare-gov/src/styles/components/_Footer.scss

```diff
@@ -1,4 +1,6 @@
-@import './Logo';
+@use '../layout' as *;
+@use './Logo' as *;
+@use '@cmsgov/design-system/src/styles/mixins' as *;
```

Comment:
did we want to do the use like this or with a relative import?

Reply:
I'd like to see if we could make an even simpler alias like `@core/mixins` or something.. gonna leave this one in as a test for when we get to compilation.

Reply:
What's wrong with an import using the package name?